weatherman/reports.py

```diff
@@ -0,0 +1,22 @@
+from weatherman.results import (YearlyExtremeWeatherResult, MonthlyAveragesWeatherResult,
+                                MonthlyExtremeBarChartResult)
```

Comment:
user `pre-commit` i can see problems in imports, that should automatically resolve, import and spacing issues according to PEP8 guidelines